Source/Fuse.Controls.Panels/Layouts/WrapLayout.uno

```diff
@@ -135,6 +153,8 @@ namespace Fuse.Layouts
 			var placements = new float4[elements.Count];
 			//minorMaxSize in each major row, assinged per element
 			var minorSizes = new float[elements.Count];
+			// save the available space for each row
+			var majorRest = new Dictionary<float, float>();
```

Comment:
This should be an array like the other tracking avraible `placements` adn `minorSizes`... or wiat, I'm not clear what this does. But a `float` should not be used as  dictionary key. Precision problems will prevent this from working.